StatisticsTests/Distributions/ParametricBootstrap.cs

```diff
@@ -37,7 +37,7 @@ public void BootstrapWorksForLP3_Test(double mean, double standardDeviation, dou
             {
                 randomNumberArray[i] = random.NextDouble();
             }
-            IDistribution testBootstrapDistribution = distributionLP3.Sample(randomNumberArray);
+            IDistribution testBootstrapDistribution = ((ContinuousDistribution)distributionLP3).Sample(randomNumberArray);
```

Comment:
does the implementation of the abstract class mean we'll need to do a lot of casting like this or is this a special case?

Reply:
we may have some. Previously this was done through the extension method. because i removed the extension method i have to leverage the abstract class.

Reply:
i could also add Sample to the IDistribution interface (and implment it on continuous distribution like i did and avoid this cast)